### PR TITLE
feat: add prev button

### PR DESCRIPTION
**Pull request by here-bossnanda**
[#5] Changelog:
- fix defect last page
- add feature prev

### CODE REVIEW

File: src/pages/QuestionPage.tsx [hunk 1]
```diff
@@ -126,52 +185,64 @@ function QuestionPage() {
...
+                        })
+                      )
+                    }}
+                  />
+                ))}
+              </div>
+            </div>
+            <div className="flex items-center gap-4">
+              <PreviousButton />
+              {questionId !== 0 && (
```
Comment (jarooda) [inline src/pages/QuestionPage.tsx:218]: use shadcn separator instead

File: src/pages/QuestionPage.tsx [hunk 2]
```diff
@@ -51,14 +65,59 @@ function QuestionPage() {
...
+
+    if (questionId > 0) {
+      navigate(`/${prevQuestionId}`);
+    }
+    }, 1000)
+  }
+
+  const PreviousButton = () => {
+    if (questionId > 0) {
+      let word = `👈 Balik ke pertanyaan ${prevQuestionId}`;
```
Comment (jarooda) [inline src/pages/QuestionPage.tsx]: make the wording general, like `Balik ke sebelumnya`

File: src/pages/QuestionPage.tsx [hunk 3]
```diff
@@ -51,14 +65,59 @@ function QuestionPage() {
...
+    } else {
+      navigate(`/${nextQuestionId}`);
+    }
     }, 1000)
   }
 
+  const goToPrevQuestion = () => {
+    setLoading(true)
+
+    setAnswers(
```
Comment (jarooda) [inline src/pages/QuestionPage.tsx]: this code block need to refactor

**Review by jarooda — CHANGES_REQUESTED** [review]

File: src/pages/QuestionPage.tsx (hunk 1 above)
Comment (here-bossnanda) [reply src/pages/QuestionPage.tsx:218]: help me to change this abang, basically i am a backend engineer :(

**Comment by jarooda** [comment]
this pull request fix #8 too

File: src/pages/QuestionPage.tsx (hunk 1 above)
Comment (jarooda) [reply src/pages/QuestionPage.tsx:218]: follow this doc https://ui.shadcn.com/docs/components/separator

File: src/pages/QuestionPage.tsx (hunk 3 above)
Comment (here-bossnanda) [reply src/pages/QuestionPage.tsx]: i just copy paste on function nextQuestion mister

**Comment by here-bossnanda** [comment]
need review again mister @jarooda

**Review by jarooda — APPROVED** [review]

**Pull request merged**